microsoft-edge/devtools-guide-chromium/memory-problems/allocation-profiler.md

```diff
@@ -21,9 +21,12 @@ ms.date: 05/04/2021
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License. -->
-# Use Allocation instrumentation on Timeline
+# Use Allocation instrumentation on timeline
 
-Use **Allocation instrumentation on timeline** to find objects that are not being properly garbage collected, and continue to retain memory.
+<!-- title in other repo:
+How to Use the Allocation Profiler Tool -->
+
+In the **Memory** tool (or _panel_), use the **Allocation instrumentation on timeline** radio button to find objects that are not being properly garbage collected, and continue to retain memory.
```

Comment:
Ah, I see you've added `or _panel_` here too. I think I'd prefer if we settled on just tool, and used it throughout, what do you think?

Reply:
outdated.  removed 'panel' from article